src/object-marker.scss

```diff
@@ -0,0 +1,98 @@
+@import "./new-settings";
+@import "./mixins";
+
+/*!
+.fd-object-marker
+*/
+$block: #{$fd-namespace}-object-marker;
+
+.#{$block} {
+  // ICON VARIABLES
+  $fd-object-marker-font-size: 1rem !default;
+  $fd-object-marker-color: var(--sapContent_MarkerIconColor) !default;
+  $fd-object-marker-text-color: var(--sapContent_LabelColor) !default;
+  $fd-object-marker-padding-left: 0.5rem !default;
+  $fd-object-marker-padding-right: 0.25rem !default;
+  // TEXT VARIABLES
+  $fd-object-marker-font-family: var(--sapFontFamily) !default;
+
+  @include fd-reset();
+
+  font-family: $fd-object-marker-font-family;
+  font-size: $fd-object-marker-font-size;
+  max-width: 100%;
+  color: $fd-object-marker-text-color;
+  word-break: break-word;
+  display: inline-flex;
+  align-items: center;
+  line-height: 1;
+
+  &::before {
+    color: $fd-object-marker-color;
+    padding: 0 $fd-object-marker-padding-right 0 0;
+  }
+
+  // CLICKABLE OBJECT STATUS
+  &--link {
+    display: inline-block;
+    text-decoration: none;
+    color: $fd-object-marker-color;
+    cursor: pointer;
+
+    &::before {
+      position: relative;
+    }
+
+    &:hover {
+      text-decoration: underline;
+
+      &::before {
+        text-decoration: none;
+      }
+    }
+
+    &:visited {
+      text-decoration: none;
+    }
+  }
+
+  // ICON ONLY MODE
+  @include fd-empty() {
+    justify-content: center;
+    padding-right: 0;
+    padding-left: 0;
+
+    &::before {
+      padding-right: 0;
+      padding-left: 0;
+    }
+  }
+
+  @include fd-rtl() {
+    &::before {
+      padding-left: $fd-object-marker-padding-right;
+      padding-right: 0;
+    }
+
+    @include fd-empty() {
+      &::before {
+        padding-right: 0;
+        padding-left: 0;
+      }
+    }
+  }
+
+  &:first-of-type + span {
+    &:empty {
+      margin-left: 0.5rem;
```

Comment:
Again, this spacing is not only when the Object Marker is an icon only  ("empty"). This is valid for ALL 2 Object markers that are adjacent. Basically if `.fd-object-marker + .fd-object-marker`, the second item (could be a span, could be a div) should have a margin-left of 0.5rem